framework/packages/abstract-std/src/native/ibc_client.rs

```diff
@@ -96,7 +96,7 @@ pub enum ExecuteMsg {
         /// host chain to be executed on
         /// Example: "osmosis"
         host_chain: TruncatedChainId,
-        funds: Vec<Coin>,
+        funds: Vec<InterchainSend>,
```

Comment:
Feel like it's worth to rename `funds` field, but not sure to what? Maybe name of `SendFunds` as well

Reply:
I'm sorry that I haven't been clear enough. I'd expect the API to be 

```rust
host_chain: TruncatedChainId,
funds: Vec<Coin>,
memo: Option<String>,
```

If memo is provided, it is attached to **every** ibc transfer msg.

Reply:
Because that is what the next version of ics20 will support!

Reply:
> I'm sorry that I haven't been clear enough. I'd expect the API to be
> 
> ```rust
> host_chain: TruncatedChainId,
> funds: Vec<Coin>,
> memo: Option<String>,
> ```
> 
> If memo is provided, it is attached to **every** ibc transfer msg.

Updated!